var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -61,6 +61,9 @@ class Openmpi(AutotoolsPackage):
     url = "https://www.open-mpi.org/software/ompi/v4.0/downloads/openmpi-4.0.0.tar.bz2"
     list_url = "http://www.open-mpi.org/software/ompi/"
 
+    version('develop', branch='master',
+        git='https://github.com/open-mpi/ompi.git')
```

Comment:
I would add this `git` attribute at the package level. That way, if someone wants to add another branch, they can.

Reply:
If I add `git` member and `version('develop', branch='master')` or even `version('develop', branch='master', git=git)` it tries to download from `url_for_version` o.0